login.go

```diff
@@ -0,0 +1,11 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func StartWebServer() {

```

Comment:
Where is port?

Reply:
you mean the port that we put in front of this  'router.Run()'  funtion?
should I use port 8080 or another port?

Reply:
You need to read a port from terminal and then run the server on that port, if no port was provided, use a default port